src/core/sys/posix/sys/stat.d

```diff
@@ -1110,6 +1110,10 @@ else version( CRuntime_Bionic )
     extern (D) bool S_ISREG( uint mode )  { return S_ISTYPE( mode, S_IFREG );  }
     extern (D) bool S_ISLNK( uint mode )  { return S_ISTYPE( mode, S_IFLNK );  }
     extern (D) bool S_ISSOCK( uint mode ) { return S_ISTYPE( mode, S_IFSOCK ); }
+
+    // Added since Lollipop
+    int utimensat(int dirfd, const char *pathname,
+        ref const(timespec)[2] times, int flags);
```

Comment:
Needed by [this code path](https://github.com/dlang/phobos/blob/6fa5e0e969bd6a4a885932e9e13dbbeda1990c1b/std/file.d#L1260), which gets [this test of higher precision on linux](https://github.com/dlang/phobos/blob/6fa5e0e969bd6a4a885932e9e13dbbeda1990c1b/std/file.d#L1339) to pass on Android.